### PR TITLE
fix: persist Experiment List sorts

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

Add `sorts` to user settings, to save sorts on new Experiment List.

## Test Plan

On new experiment list (`f_explist_v2=on`)
- [ ] Set sorting on multiple columns
- [ ] Refresh page and make sure sorts that were set persist

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by keita-determined** [comment]
question: do we wanna keep param query of sort in URL? Even if user modifies param query, useSettings doesn't take it. It looks a bit redundant to keep it to me

**Comment by ClaireNeveu** [comment]
@keita-determined The ability to share your view (which is what the URL serialization would accomplish) is explicitly out of scope. Early on in the project we considered a bunch of ways to handle sharing and didn't find a clear winner so we're leaving it out of V1.

File: webui/react/src/pages/F_ExpList/F_ExperimentList.tsx [hunk 1]
```diff
@@ -186,15 +186,26 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   const onSortChange = useCallback(
     (sorts: Sort[]) => {
       setSorts(sorts);
+      updateSettings({
+        // save sorts to settings:
+        sorts: JSON.stringify(sorts),
+      });
```
Comment (ashtonG) [inline webui/react/src/pages/F_ExpList/F_ExperimentList.tsx:192]: consider reading/writing the `sortString` instead -- it only keeps the valid sort entries.

**Comment by keita-determined** [comment]
What i meant is that  we can remove the query param which is defined https://github.com/determined-ai/determined/blob/main/webui/react/src/pages/F_ExpList/F_ExperimentList.tsx#L65. Especially @johnkim-det set `skipUrlEncoding: true`.

**Comment by johnkim-det** [comment]
I'll go ahead and remove the code that reads `sort` from `searchParams`.

As far as saving `sorts` vs `sortString`: I think we would want to set both based on saved settings, which could be achieved with saving either value, but looks like saving `sorts` and building `sortString` from it is consistent with what `onSortChange` already does.

**Pull request merged**